frontend/src/pages/LoginAndRegisterPage.module.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap');
+
+body{
+    margin: 0;
+}
+
+.mainContainer{
+    height: 100vh;
+    width: 100vw;
+    background: url('../images/background.jpg');
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.formBlock{
+    height:70vh;
+    width: 50vw;
+    background-color:rgba(5, 4, 3, 0.5);
+    border-radius: 20px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.formBlock a{
+    color:#0FFBC4;
+    font-family: "Inter", sans-serif;
+    font-weight: 400;
+    font-size: 24px;
+    text-decoration: underline;
+
+}
+form{
+    height: 40vh;
+    width: 50vw;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+}
+.formBlock h1{
+    font-family: "Inter", sans-serif;
+    font-size: 60px;
+    font-weight: 800;
+    color: #00FFF1;
+    text-shadow: 0 4px 7px rgba(15, 251, 196, 0.5);
+}
+input{
+    width: 45%;
+    height: 20%;
+    padding-left: 2vw;
+    background-color: #14B8AF;
+    color: #00263B;
+    font-size: 24px;
+    border: none;
+    border-radius: 16px;
+    box-shadow: 0 4px 7px rgba(15, 251, 196, 0.5);
+    box-sizing: border-box;
```

Comment:
this box sizing should be enabled by default on all elements. That's why we add it to the snippet I've sent you earlier.